data/assets/examples/renderable/renderablenodearrow/nodearrow_relativevalues.asset

```diff
@@ -0,0 +1,40 @@
+-- Relative Units for Offset and Length
+-- This example shows an arrow pointing from one scene graph node in the direction of
+-- another, but where the size is specified using relative values (based on the bounding
+-- sphere of the start node). Here it points from Earth to the Moon.
+--
+-- Note that the arrows are generated as objects in 3D space and need to have a size
+-- that is suitable for the scene graph nodes they refer to. Here it is set based on
+-- the size of the start node, i.e. Earth.
+
+local earth = asset.require("scene/solarsystem/planets/earth/earth")
+local moon = asset.require("scene/solarsystem/planets/earth/moon/moon")
+
+local Node = {
+  Identifier = "RenderableNodeArrow_Example_Relative",
+  Parent = earth.Earth.Identifier,
```

Comment:
But here you can consider adding the clarifying comment as used before to explain what a parent node is again. Or do we assume the user already knows this by the time they reach this example?

Reply:
No, the comment should be here as well. Good 👀